src/_codux/boards/header/header.board.tsx

```diff
@@ -0,0 +1,14 @@
+import { createBoard } from '@wixc3/react-board';
+import { Header } from '../../../components/header';
+
+export default createBoard({
+   name: 'Header',
+   Board: () => <Header />,
```

Comment:
Notice the type of `HeaderProps` requires `cardCount` so this is marked as an error there.